.github/workflows/integrity-check.yml

```diff
@@ -53,8 +53,15 @@ jobs:
       - run: npm run lint
       # runs tests in node environment without attempting to generate code coverage badges
       - run: npm run test:node:ci
+
+      - name: Upload coverage reports to Codecov
```

Comment:
@0scvr Looks like CI is failing with this error.
`Unable to locate build via Github Actions API. Please upload with the Codecov repository upload token to resolve issue.`

We probably need to get the codecov secret like you were doing in your last commit, but I don't think secret isn't set up for this repo. We'll get that set up for you. Since it's afternoon on a Friday, we might not have that set up until early next week.

@frankhinek when you see this, let's sync and get the codecov secret set up for dwn-sdk-js.

Reply:
I've gone ahead and created the secret myself.